app/src/main/java/org/mozilla/fenix/utils/ManufacturerCodes.kt

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.utils
+
+import android.os.Build
+import java.util.Locale
+
+/**
+ * A listing of codes returned by [android.os.Build.MANUFACTURER] for different manufacturers.
+ * It may be good to convert to lower case before doing comparison.
+ */
+object ManufacturerCodes {
+    const val HUAWEI: String = "HUAWEI"
+    const val LG = "LGE"
+    const val ONE_PLUS: String = "OnePlus"
+    const val SAMSUNG = "samsung"
+
+    val isLG get() = manufacturerLower == LG
```

Comment:
This will always return false. maybe we could use `equals` with `ignoreCase` = true. And there will be no need for `manufacturerLower `

Reply:
Same for `isSamsung `

Reply:
Good catch! I'm updating the `ManufacturerCodes` commit to fix the error and adding a test so I can verify these functions actually work 😅 

> maybe we could use equals with ignoreCase = true

Oh neat, I didn't realize that function existed